ocrd_cis/ocropy/binarize.py

```diff
@@ -1,34 +1,27 @@
 from __future__ import absolute_import
+from logging import Logger
 
-import os.path
 import cv2
 import numpy as np
 from PIL import Image
+from os.path import abspath, dirname, join
+
+from typing import Tuple
 
 #import kraken.binarization
 
 from ocrd_utils import (
     getLogger,
     make_file_id,
-    assert_file_grp_cardinality,
     MIMETYPE_PAGE
 )
-from ocrd_modelfactory import page_from_file
-from ocrd_models.ocrd_page import (
-    to_xml, AlternativeImageType
-)
+from ocrd_models.ocrd_page import AlternativeImageType, OcrdPage, to_xml
 from ocrd import Processor
 
-from .. import get_ocrd_tool
 from . import common
-from .common import (
-    pil2array, array2pil,
-    # binarize,
-    remove_noise)
-
-#sys.path.append(os.path.dirname(os.path.abspath(__file__)))
+from .common import array2pil, determine_zoom, pil2array, remove_noise
 
-TOOL = 'ocrd-cis-ocropy-binarize'
+#sys.path.append(dirname(abspath(__file__)))
```

Comment:
```suggestion
```

Reply:
I did not want to remove any previously commented-out code other than just to optimize the imports.